projects/ngx-row-accordion/src/lib/ngx-row-accordion.component.ts

```diff
@@ -21,7 +23,7 @@ export class NgxRowAccordionComponent implements OnInit, OnDestroy {
   private id: string = uuid();
   private onDestroy$: Subject<void> = new Subject();
 
-  constructor(private ngxRowAccordionService: NgxRowAccordionService) {}
+  public status = false;
```

Comment:
`status` is IMO a bit too generic. Could you rename to something more specific?
Maybe `isPanelOpened`?